lua/luapad/evaluator.lua

```diff
@@ -102,6 +105,11 @@ function Evaluator:eval()
   self.output = {}
 
   local code = vim.api.nvim_buf_get_lines(self.buf, 0, -1, {})
+  local benchmark_line, benchmark_line_num = table_find(code, function (line)
+    return line:match("-- #luapad:benchmark") ~= nil
```

Comment:
```lua
return line:match("^ *%-%- *#luapad:benchmark")
```
The `-` char, has actually special meaning in lua pattern matching. Yeah, I know, tricky! It's a non-greedy version of `*`.